.env.example

```
@@ -79,6 +92,30 @@ CODEX_ACCOUNT_ID=
+# content, and archon reports `dag.node_empty_output` after ~150 s. Fresh installs that
+# copy `.env.example → .env` lose this fix and silently regress.
+#
+# `AIDERDESK_PROJECT_DIR_REMAP` fixes this — `translateProjectDir`
+# (packages/providers/src/community/aiderdesk/provider.ts, sendQuery step 1a, commit
+# de175bc) consumes the JSON and forwards the host path. Array form preferred (declaration
+# order = match order, prefix-matching, regex `from` → literal `to`). Operational
+# precedence, top-down highest first: requestOptions.env > process.env > assistantConfig
+# > identity. DO NOT quote the JSON — the value is parsed verbatim by
+# `parseProjectDirRemapJson`.
+AIDERDESK_API_URL=http://172.18.0.1:24337
+AIDERDESK_PROJECT_DIR_REMAP=[{"from":"^/app($|/)", "to":"/home/lfontanez/dev/archon-v2"}, {"from":"^/host/projects/", "to":"/home/lfontanez/dev/"}]  # JSON: array form (regex `from` → literal `to`); longest match / declaration order. DO NOT quote the JSON.
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The AIDERDESK_PROJECT_DIR_REMAP key is not assigned properly

(SubstitutionKey)